mypy/checkmember.py

```diff
@@ -180,6 +183,7 @@ def analyze_member_var_access(name: str, itype: Instance, info: TypeInfo,
     This is logically part of analyze_member_access and the arguments are
     similar.
     """
```

Comment:
Yes, I have commented about it above. Should I rename it to `actual_self`? Or perhaps report_type and actual_self just happen to coincide and they are actually different things.